.rubocop.yml

```diff
@@ -100,6 +101,68 @@ Packaging/RequireHardcodingLib:
 Packaging/RequireRelativeHardcodingLib:
   Enabled: true
 
+Performance:
+  Enabled: true
+
+Performance/AncestorsInclude:
+  Enabled: true
+
+Performance/BigDecimalWithNumericArgument:
+  Enabled: true
+
+Performance/BlockGivenWithExplicitBlock:
+  Enabled: true
+
+Performance/CollectionLiteralInLoop:
+  Enabled: true
+  Exclude:
+    - spec/**/*
+
+Performance/ConcurrentMonotonicTime:
+  Enabled: true
+
+Performance/ConstantRegexp:
+  Enabled: true
+
+Performance/MapCompact:
+  Enabled: true
+
+Performance/MapMethodChain:
+  Enabled: true
+
+Performance/MethodObjectAsBlock:
+  Enabled: true
+
+Performance/RedundantEqualityComparisonBlock:
+  Enabled: true
+
+Performance/RedundantSortBlock:
+  Enabled: true
+
+Performance/RedundantSplitRegexpArgument:
+  Enabled: true
+
+Performance/RedundantStringChars:
+  Enabled: true
+
+Performance/ReverseFirst:
+  Enabled: true
+
+Performance/SortReverse:
+  Enabled: true
+
+Performance/Squeeze:
+  Enabled: true
+
+Performance/StringIdentifierArgument:
+  Enabled: true
+
+Performance/StringInclude:
+  Enabled: true
+
+Performance/Sum:
+  Enabled: true
```

Comment:
Please change the enabled and option values here as needed. I just copied a config from elsewhere that has the full list of cops so you don't have to write out each one manually.

```suggestion
Performance/AncestorsInclude:
  Enabled: false

Performance/ArraySemiInfiniteRangeSlice:
  Enabled: false

Performance/BigDecimalWithNumericArgument:
  Enabled: true

Performance/BindCall:
  Enabled: true

Performance/BlockGivenWithExplicitBlock:
  Enabled: true

Performance/Caller:
  Enabled: true

Performance/CaseWhenSplat:
  Enabled: true

Performance/Casecmp:
  Enabled: false

Performance/ChainArrayAllocation:
  Enabled: false

Performance/CollectionLiteralInLoop:
  Enabled: true

Performance/CompareWithBlock:
  Enabled: true

Performance/ConcurrentMonotonicTime:
  Enabled: true

Performance/ConstantRegexp:
  Enabled: true

Performance/Count:
  Enabled: true

Performance/DeletePrefix:
  Enabled: true

Performance/DeleteSuffix:
  Enabled: true

Performance/Detect:
  Enabled: true

Performance/DoubleStartEndWith:
  Enabled: true
  IncludeActiveSupportAliases: true

Performance/EndWith:
  Enabled: true

Performance/FixedSize:
  Enabled: true

Performance/FlatMap:
  Enabled: true
  EnabledForFlattenWithoutParams: false

Performance/InefficientHashSearch:
  Enabled: true

Performance/IoReadlines:
  Enabled: true

Performance/MapCompact:
  Enabled: false

Performance/MapMethodChain:
  Enabled: false

Performance/MethodObjectAsBlock:
  Enabled: true

Performance/OpenStruct:
  Enabled: true

Performance/RangeInclude:
  Enabled: true

Performance/RedundantBlockCall:
  Enabled: false

Performance/RedundantEqualityComparisonBlock:
  Enabled: false

Performance/RedundantMatch:
  Enabled: true

Performance/RedundantMerge:
  Enabled: true
  MaxKeyValuePairs: 2

Performance/RedundantSortBlock:
  Enabled: true

Performance/RedundantSplitRegexpArgument:
  Enabled: true

Performance/RedundantStringChars:
  Enabled: true

Performance/RegexpMatch:
  Enabled: true

Performance/ReverseEach:
  Enabled: true

Performance/ReverseFirst:
  Enabled: true

Performance/SelectMap:
  Enabled: false

Performance/Size:
  Enabled: true

Performance/SortReverse:
  Enabled: true

Performance/Squeeze:
  Enabled: true

Performance/StartWith:
  Enabled: true

Performance/StringIdentifierArgument:
  Enabled: true

Performance/StringInclude:
  Enabled: true

Performance/StringReplacement:
  Enabled: true

Performance/Sum:
  Enabled: false

Performance/TimesMap:
  Enabled: true

Performance/UnfreezeString:
  Enabled: true

Performance/UriDefaultParser:
  Enabled: true
```